backend/v3/storage/database/operators.go

```diff
@@ -137,6 +137,6 @@ const (
 
 func writeBooleanOperation[T Boolean](builder *StatementBuilder, col Column, value T) {
 	col.Write(builder)
-	builder.WriteString(" IS ")
+	builder.WriteString(" = ")
```

Comment:
Just take care with this change:

```sql
select NULL IS NULL; -- true
select NULL = NULL; -- NULL
```

Not sure if you depend somewhere on `NULL` equality, but that would be broken by this change.

Reply:
there is a IsNull condition for such cases: https://github.com/zitadel/zitadel/blob/866c2971f1a76557df4dd3a1f6b9547788a6518c/backend/v3/storage/database/condition.go#L70